### PR TITLE
Fix ansible config

### PR DESCRIPTION
**Pull request by Spacelocust**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #(issue) | No issue linked.

## Type of change

Please add any relevant information to the type of change you've made.

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update (or is a documentation update)
-   [x] HIGH IMPORTANCE (requires a fast merging)

## Has This Been Tested?

Please describe the tests that you ran to verify your changes. It is way more likely for a PR to be merged and approved with successful tests.

-   [x] Yes, I ran tests
-   [ ] No, I did not include any test
-   [ ] No test is needed for this change

### CODE REVIEW

File: .github/workflows/deploy-vps.yml [hunk 1]
```diff
@@ -38,3 +38,5 @@ jobs:
 
             [prod:vars]
             ansible_python_interpreter=/usr/bin/python3
+          options: |
+            -vv
```
Comment (YummYume) [inline .github/workflows/deploy-vps.yml]:
```suggestion
            -vv

```

File: .github/workflows/deploy-vps.yml [hunk 2]
```diff
@@ -13,12 +13,12 @@ jobs:
...
-        with:
-          path: /var/cache/apt/archives
-          key: ${{ runner.os }}-apt-${{ hashFiles('**/apt.lock') }}
-          restore-keys: ${{ runner.os }}-apt-
+      # - name: Cache APT packages
+      #   uses: actions/cache@v2
+      #   with:
+      #     path: /var/cache/apt/archives
+      #     key: ${{ runner.os }}-apt-${{ hashFiles('**/apt.lock') }}
+      #     restore-keys: ${{ runner.os }}-apt-
```
Comment (YummYume) [inline .github/workflows/deploy-vps.yml]: Supprimer?

File: ansible/create-env.yml [hunk 3]
```diff
@@ -24,7 +25,7 @@
   ansible.builtin.copy:
     dest: ./app/api/.env.prod
     force: true
-    mode: "0660"
+    mode: "0664"
```
Comment (YummYume) [inline ansible/create-env.yml]: Il faudrait pas donner les droits plutôt que d'autoriser tout le monde a lire ?

File: compose.prod.yml [hunk 4]
```diff
@@ -104,3 +104,8 @@ services:
       start_period: 60s
     networks:
       - rabbitmq
+
+networks:
+  webserver:
+  rabbitmq:
+  database:
```
Comment (YummYume) [inline compose.prod.yml]:
```suggestion
  database:

```

ca marche si on donne juste les noms?

```yaml
networks:
  - webserver
  - rabbitmq
  - database
```

File: ansible/create-env.yml (hunk 3 above)
Comment (Spacelocust) [reply ansible/create-env.yml]: je vais check dans la doc

File: compose.prod.yml (hunk 4 above)
Comment (Spacelocust) [reply compose.prod.yml]: Nn j'ai essayé, il s'attend à avoir un objet

**Review by YummYume — DISMISSED** [review]

**Review by YummYume — APPROVED** [review]

**Pull request merged**